app/src/processing/app/Base.java

```diff
@@ -1525,9 +1524,11 @@ public void paint(Graphics g) {
           g2.setRenderingHint(RenderingHints.KEY_TEXT_ANTIALIASING,
                               RenderingHints.VALUE_TEXT_ANTIALIAS_OFF);
 
-          g.setFont(new Font("SansSerif", Font.PLAIN, 11));
+          int scale = Theme.getInteger("gui.scalePercent");
+          Font f = new Font("SansSerif", Font.PLAIN, 11 * scale / 100);
+          g.setFont(f);
           g.setColor(Color.white);
-          g.drawString(BaseNoGui.VERSION_NAME, 50, 30);
+          g.drawString(BaseNoGui.VERSION_NAME, 50 * scale / 100, 30 * scale / 100);
```

Comment:
@PaulStoffregen PR doesn't merge any more because I've changed splashscreen and had to change version string position
When resolving merge, take your version and change
`g.drawString(BaseNoGui.VERSION_NAME, 50 * scale / 100, 30 * scale / 100);`
to
`g.drawString(BaseNoGui.VERSION_NAME, 33 * scale / 100, 20 * scale / 100);`